Tests/NIOPosixTests/SyscallAbstractionLayer.swift

```diff
@@ -884,7 +884,18 @@ extension SALTest {
             }
         }
     }
-
+    
+    func assertBindFailure(expectedAddress: SocketAddress, file: StaticString = #filePath, line: UInt = #line) throws {
```

Comment:
Can we fold this into the existing `assertBind` call, with an optional `errorReturn` parameter that defaults to `nil`? That's a little closer to how we typically structure things in the SAL.

Reply:
done!